base/linalg/bunchkaufman.jl

```diff
@@ -153,43 +153,43 @@ permutation:
  3
  2
 
-julia> F[:L]*F[:D]*F[:L]' - A[F[:p], F[:p]]
+julia> F.L*F.D*F.L' - A[F.p, F.p]
 3×3 Array{Float64,2}:
  0.0  0.0  0.0
  0.0  0.0  0.0
  0.0  0.0  0.0
 
 julia> F = bkfact(Symmetric(A));
 
-julia> F[:U]*F[:D]*F[:U]' - F[:P]*A*F[:P]'
+julia> F.U*F.D*F.U' - F.P*A*F.P'
 3×3 Array{Float64,2}:
  0.0  0.0  0.0
  0.0  0.0  0.0
  0.0  0.0  0.0
 ```
 """
-function getindex(B::BunchKaufman{T}, d::Symbol) where {T<:BlasFloat}
+@inline function getproperty(B::BunchKaufman{T}, d::Symbol) where {T<:BlasFloat}
```

Comment:
this seems like a lot to inline

Reply:
I think inlining is necessary to get the constant propagation working sufficiently well to make the factors inferred.

Reply:
Hm. Maybe that wasn't the issue. It seems to work now. I'll try to remove the annotations and see what happens.